### ISSUE TITLE
PDOException in imaging browser

### ISSUE THREAD

**Issue by maximemulder** [aces/Loris#9500]
Got this error on main while accessing the imaging browser / view session page. No time to debug it now, but that should probably be done until the next release.

```
 Fatal error: Uncaught PDOException: SQLSTATE[HY000]: General error: 2014 Cannot execute queries while there are pending result sets. Consider unsetting the previous PDOStatement or calling PDOStatement::closeCursor() in /var/www/loris/src/Database/Query.php:42 Stack trace: #0 /var/www/loris/src/Database/Query.php(42): PDOStatement->execute() #1 /var/www/loris/src/Database/Query.php(33): LORIS\Database\Query->getIterator() #2 /var/www/loris/php/libraries/Database.class.inc(840): LORIS\Database\Query->getFirstRow() #3 /var/www/loris/php/libraries/Database.class.inc(979): Database->pselectRow() #4 /var/www/loris/php/libraries/Database.class.inc(1008): Database->pselectOne() #5 /var/www/loris/modules/imaging_browser/php/viewsession.class.inc(397): Database->pselectOneInt() #6 /var/www/loris/modules/imaging_browser/php/viewsession.class.inc(310): LORIS\imaging_browser\ViewSession->_getManualCaveatViolationsResolvedID() #7 /var/www/loris/modules/imaging_browser/php/viewsession.class.inc(116): LORIS\imaging_browser\ViewSession->_setFilesData() #8 /var/www/loris/php/libraries/NDB_Form.class.inc(133): LORIS\imaging_browser\ViewSession->setup() #9 /var/www/loris/src/Middleware/UserPageDecorationMiddleware.php(248): NDB_Form->handle() #10 /var/www/loris/src/Middleware/PageDecorationMiddleware.php(59): LORIS\Middleware\UserPageDecorationMiddleware->process() #11 /var/www/loris/php/libraries/NDB_Page.class.inc(726): LORIS\Middleware\PageDecorationMiddleware->process() #12 /var/www/loris/php/libraries/Module.class.inc(322): NDB_Page->process() #13 /var/www/loris/src/Middleware/ResponseGenerator.php(51): Module->handle() #14 /var/www/loris/src/Middleware/AuthMiddleware.php(64): LORIS\Middleware\ResponseGenerator->process() #15 /var/www/loris/src/Router/ModuleRouter.php(75): LORIS\Middleware\AuthMiddleware->process() #16 /var/www/loris/src/Middleware/ExceptionHandlingMiddleware.php(54): LORIS\Router\ModuleRouter->handle() #17 /var/www/loris/src/Router/BaseRouter.php(138): LORIS\Middleware\ExceptionHandlingMiddleware->process() #18 /var/www/loris/src/Middleware/ResponseGenerator.php(51): LORIS\Router\BaseRouter->handle() #19 /var/www/loris/src/Middleware/ContentLength.php(53): LORIS\Middleware\ResponseGenerator->process() #20 /var/www/loris/htdocs/index.php(78): LORIS\Middleware\ContentLength->process() #21 {main} thrown in /var/www/loris/src/Database/Query.php on line 42
```

**Comment by maximemulder** [comment]
@cmadjar I think that means that my MRI scan type PR is not at fault regarding this bug !

**Comment by driusan** [comment]
I am not able to reproduce this. I just accessed a random assortment of candidates on Raisinbread. Is there a specific candidate you're trying to access?

**Comment by driusan** [comment]
Nevermind, I reproduced it on SessionID 2144. I'll look into this.

**Comment by driusan** [comment]
This should be fixed by #9501, can you test it?
